<html xmlns:v="urn:schemas-microsoft-com:vml"
xmlns:o="urn:schemas-microsoft-com:office:office"
xmlns:w="urn:schemas-microsoft-com:office:word"
xmlns:m="http://schemas.microsoft.com/office/2004/12/omml"
xmlns="http://www.w3.org/TR/REC-html40">

<head>
<meta http-equiv=Content-Type content="text/html; charset=windows-1252">
<meta name=ProgId content=Word.Document>
<meta name=Generator content="Microsoft Word 12">
<meta name=Originator content="Microsoft Word 12">
<link rel=File-List href="BNW_workflow_test_files/filelist.xml">
<link rel=Edit-Time-Data href="BNW_workflow_test_files/editdata.mso">
<link rel=themeData href="BNW_workflow_test_files/themedata.thmx">
<link rel=colorSchemeMapping
href="BNW_workflow_test_files/colorschememapping.xml">
<style>
<!--
 /* Font Definitions */
 @font-face
	{font-family:"Cambria Math";
	panose-1:2 4 5 3 5 4 6 3 2 4;
	mso-font-charset:1;
	mso-generic-font-family:roman;
	mso-font-format:other;
	mso-font-pitch:variable;
	mso-font-signature:0 0 0 0 0 0;}
@font-face
	{font-family:Calibri;
	panose-1:2 15 5 2 2 2 4 3 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-1610611985 1073750139 0 0 159 0;}
@font-face
	{font-family:Tahoma;
	panose-1:2 11 6 4 3 5 4 4 2 4;
	mso-font-charset:0;
	mso-generic-font-family:swiss;
	mso-font-pitch:variable;
	mso-font-signature:-520081665 -1073717157 41 0 66047 0;}
 /* Style Definitions */
 p.MsoNormal, li.MsoNormal, div.MsoNormal
	{mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-parent:"";
	margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:0in;
	line-height:115%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
p.MsoAcetate, li.MsoAcetate, div.MsoAcetate
	{mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-link:"Balloon Text Char";
	margin:0in;
	margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:8.0pt;
	font-family:"Tahoma","sans-serif";
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;}
p.MsoListParagraph, li.MsoListParagraph, div.MsoListParagraph
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:.5in;
	mso-add-space:auto;
	line-height:115%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
p.MsoListParagraphCxSpFirst, li.MsoListParagraphCxSpFirst, div.MsoListParagraphCxSpFirst
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	margin-bottom:.0001pt;
	mso-add-space:auto;
	line-height:115%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
p.MsoListParagraphCxSpMiddle, li.MsoListParagraphCxSpMiddle, div.MsoListParagraphCxSpMiddle
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0in;
	margin-right:0in;
	margin-bottom:0in;
	margin-left:.5in;
	margin-bottom:.0001pt;
	mso-add-space:auto;
	line-height:115%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
p.MsoListParagraphCxSpLast, li.MsoListParagraphCxSpLast, div.MsoListParagraphCxSpLast
	{mso-style-priority:34;
	mso-style-unhide:no;
	mso-style-qformat:yes;
	mso-style-type:export-only;
	margin-top:0in;
	margin-right:0in;
	margin-bottom:10.0pt;
	margin-left:.5in;
	mso-add-space:auto;
	line-height:115%;
	mso-pagination:widow-orphan;
	font-size:11.0pt;
	font-family:"Calibri","sans-serif";
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
span.BalloonTextChar
	{mso-style-name:"Balloon Text Char";
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-unhide:no;
	mso-style-locked:yes;
	mso-style-link:"Balloon Text";
	mso-ansi-font-size:8.0pt;
	mso-bidi-font-size:8.0pt;
	font-family:"Tahoma","sans-serif";
	mso-ascii-font-family:Tahoma;
	mso-hansi-font-family:Tahoma;
	mso-bidi-font-family:Tahoma;}
.MsoChpDefault
	{mso-style-type:export-only;
	mso-default-props:yes;
	font-size:10.0pt;
	mso-ansi-font-size:10.0pt;
	mso-bidi-font-size:10.0pt;
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-fareast-font-family:Calibri;
	mso-fareast-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;
	mso-bidi-font-family:"Times New Roman";
	mso-bidi-theme-font:minor-bidi;}
@page WordSection1
	{size:8.5in 11.0in;
	margin:1.0in 1.0in 1.0in 1.0in;
	mso-header-margin:.5in;
	mso-footer-margin:.5in;
	mso-paper-source:0;}
div.WordSection1
	{page:WordSection1;}
 /* List Definitions */
 @list l0
	{mso-list-id:779757819;
	mso-list-type:hybrid;
	mso-list-template-ids:939584274 67698705 67698713 67698715 67698703 67698713 67698715 67698703 67698713 67698715;}
@list l0:level1
	{mso-level-text:"%1\)";
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l0:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1
	{mso-list-id:1813714603;
	mso-list-type:hybrid;
	mso-list-template-ids:939584274 67698705 67698713 67698715 67698703 67698713 67698715 67698703 67698713 67698715;}
@list l1:level1
	{mso-level-text:"%1\)";
	mso-level-tab-stop:none;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level2
	{mso-level-tab-stop:1.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level3
	{mso-level-tab-stop:1.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level4
	{mso-level-tab-stop:2.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level5
	{mso-level-tab-stop:2.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level6
	{mso-level-tab-stop:3.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level7
	{mso-level-tab-stop:3.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level8
	{mso-level-tab-stop:4.0in;
	mso-level-number-position:left;
	text-indent:-.25in;}
@list l1:level9
	{mso-level-tab-stop:4.5in;
	mso-level-number-position:left;
	text-indent:-.25in;}
ol
	{margin-bottom:0in;}
ul
	{margin-bottom:0in;}
-->
</style>
<!--[if gte mso 10]>
<style>
 /* Style Definitions */
 table.MsoNormalTable
	{mso-style-name:"Table Normal";
	mso-tstyle-rowband-size:0;
	mso-tstyle-colband-size:0;
	mso-style-noshow:yes;
	mso-style-priority:99;
	mso-style-qformat:yes;
	mso-style-parent:"";
	mso-padding-alt:0in 5.4pt 0in 5.4pt;
	mso-para-margin:0in;
	mso-para-margin-bottom:.0001pt;
	mso-pagination:widow-orphan;
	font-size:10.0pt;
	font-family:"Calibri","sans-serif";
	mso-ascii-font-family:Calibri;
	mso-ascii-theme-font:minor-latin;
	mso-hansi-font-family:Calibri;
	mso-hansi-theme-font:minor-latin;}
</style>
<![endif]--><!--[if gte mso 9]><xml>
 <o:shapedefaults v:ext="edit" spidmax="4098"/>
</xml><![endif]--><!--[if gte mso 9]><xml>
 <o:shapelayout v:ext="edit">
  <o:idmap v:ext="edit" data="1"/>
 </o:shapelayout></xml><![endif]-->
</head>

<body lang=EN-US style='tab-interval:.5in'>

<div class=WordSection1>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'>This workflow will provide
an overview of how to use BNW to:<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpFirst style='margin-right:307.5pt;mso-add-space:
auto;text-indent:-.25in;mso-list:l1 level1 lfo2'><![if !supportLists]><span
style='font-size:12.0pt;line-height:115%;font-family:"Arial","sans-serif";
mso-fareast-font-family:Arial'><span style='mso-list:Ignore'>1)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:115%;font-family:"Arial","sans-serif"'>Select
a structure learning method and upload data into BNW<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpMiddle style='margin-right:307.5pt;mso-add-space:
auto;text-indent:-.25in;mso-list:l1 level1 lfo2'><![if !supportLists]><span
style='font-size:12.0pt;line-height:115%;font-family:"Arial","sans-serif";
mso-fareast-font-family:Arial'><span style='mso-list:Ignore'>2)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:115%;font-family:"Arial","sans-serif"'>Use
the BNW structural restraint interface<o:p></o:p></span></p>

<p class=MsoListParagraphCxSpLast style='margin-right:307.5pt;mso-add-space:
auto;text-indent:-.25in;mso-list:l1 level1 lfo2'><![if !supportLists]><span
style='font-size:12.0pt;line-height:115%;font-family:"Arial","sans-serif";
mso-fareast-font-family:Arial'><span style='mso-list:Ignore'>3)<span
style='font:7.0pt "Times New Roman"'>&nbsp;&nbsp;&nbsp; </span></span></span><![endif]><span
style='font-size:12.0pt;line-height:115%;font-family:"Arial","sans-serif"'>Make
predictions with the network structure<o:p></o:p></span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:14.0pt;line-height:115%;font-family:"Arial","sans-serif"'>1.
Select structure learning method and uploading data into BNW<o:p></o:p></span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'>For this workflow, we are
creating a network using a biological data set containing both continuous and
discrete variables. The variables are a genotype (the discrete variable), three
gene expression traits (Gene1, Gene2, Gene3), and a phenotype.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'>We do not know the network
structure for this dataset, so we will first use BNW to learn the structure. Selecting
<u>Learn a network model from data</u> on the BNW home page presents a list of the
three structure learning methods that are currently implemented in BNW. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><!--[if gte vml 1]><v:shapetype
 id="_x0000_t75" coordsize="21600,21600" o:spt="75" o:preferrelative="t"
 path="m@4@5l@4@11@9@11@9@5xe" filled="f" stroked="f">
 <v:stroke joinstyle="miter"/>
 <v:formulas>
  <v:f eqn="if lineDrawn pixelLineWidth 0"/>
  <v:f eqn="sum @0 1 0"/>
  <v:f eqn="sum 0 0 @1"/>
  <v:f eqn="prod @2 1 2"/>
  <v:f eqn="prod @3 21600 pixelWidth"/>
  <v:f eqn="prod @3 21600 pixelHeight"/>
  <v:f eqn="sum @0 0 1"/>
  <v:f eqn="prod @6 1 2"/>
  <v:f eqn="prod @7 21600 pixelWidth"/>
  <v:f eqn="sum @8 21600 0"/>
  <v:f eqn="prod @7 21600 pixelHeight"/>
  <v:f eqn="sum @10 21600 0"/>
 </v:formulas>
 <v:path o:extrusionok="f" gradientshapeok="t" o:connecttype="rect"/>
 <o:lock v:ext="edit" aspectratio="t"/>
</v:shapetype><v:shape id="Picture_x0020_2" o:spid="_x0000_i1037" type="#_x0000_t75"
 style='width:300pt;height:117pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="BNW_workflow_test_files/image003.jpg" o:title="" croptop="19333f"
  cropbottom="34461f" cropleft="11633f" cropright="33260f"/>
</v:shape><![endif]--><![if !vml]><img width=450 height=156
src="BNW_workflow_test_files/image003.jpg" v:shapes="Picture_x0020_2"><![endif]></span><span
style='font-size:12.0pt;line-height:115%;font-family:"Arial","sans-serif"'><o:p></o:p></span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'>Because the network contains
only 5 variables, we can use any of these structure learning methods. If more
than 6 variables are present, using the global optimal search method would likely be necessary.
We will use the exhaustive search with model averaging method in this workflow.
More information about the structure learning methods can be found on the BNW
help page.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'>After selecting the
Exhaustive Search button, we are prompted to upload a file containing the data.
<o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>
<img width=700 height=225
src="BNW_workflow_test_files/image006.jpg" v:shapes="Picture_x0020_5"><o:p></o:p></span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><br>The figure
below shows BNW after we have uploaded the data file. The first line of the
data contains the names of the variables included in the network, while the
remaining lines are the variable values for individual samples. A description
of how to format data files for use in BNW can be found on the BNW help page and can be accessed by the the <u>Data formatting guidelines</u> option in the left menu. Additionally, two additional options, <u>Select additional constraints</u> or <u>Perform Bayesian network modeling with no restraints</u>, appear in the left menu after loading the data file.<o:p></o:p></span></p><br>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>
<![if !vml]><img width=675 height=225
src="BNW_workflow_test_files/image008.jpg" v:shapes="Picture_x0020_8"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:14.0pt;line-height:115%;font-family:"Arial","sans-serif"'>2.
Use the structural constraint interface<o:p></o:p></span></b></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>Here, we will use the
structural constraint interface to help identify biologically meaningful
network structures. Specifically, we want to investigate how the genotype
impacts gene expression which then impacts the phenotype.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>The first
section of the structural constraint interface allows users to assign the variables
(nodes) in the network to tiers. By default, three tiers are shown, but this
can be changed by selecting a different number in the drop-down menu. The
leftmost box of this section contains draggable boxes with the names of the
variables in the network. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>
<![if !vml]><img width=750 height=300
src="BNW_workflow_test_files/image010.jpg" v:shapes="Picture_x0020_14"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-top:0in;margin-right:307.5pt;margin-bottom:
0in;margin-left:0in;margin-bottom:.0001pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>For this
network, we want to assign Genotype to Tier1, the gene expression traits to
Tier2, and the Phenotype to Tier3.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>
<![if !vml]><img width=750 height=300
src="BNW_workflow_test_files/image012.jpg" v:shapes="Picture_x0020_17"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>The second
section in the structural constraint interface allows users to specify the
types of interactions that are allowed both between and within tiers. In this
case, we will keep the default settings, which will allow there to be edges
between nodes within a tier, prevent nodes in Tier1 from being the child of
Tier2 and Tier3 nodes, and prevent nodes in Tier2 from being the child of Tier2
nodes. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><![if !vml]><img width=475 height=200
src="BNW_workflow_test_files/image014.jpg" v:shapes="Picture_x0020_20"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>The final
section of the structural constraint interface would allow for the
specification of particular edges that should be banned from the network or
required to be in the network. Users can identify these edges by dragging the
nodes to appropriate boxes. In this case, we do not want to ban or require any
specific edges. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>
<![if !vml]><img width=430 height=227
src="BNW_workflow_test_files/image016.jpg" v:shapes="Picture_x0020_23"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>After
entering the structural constraints, BNW will perform structure learning from
the data after clicking <u>Perform Bayesin network modeling</u> on the left menu and
present the structure of the network as shown below. Genotype, the discrete
node, is shown as a bar chart with the bars showing the fraction of samples
with each genotype in the data, while the other nodes are shown as lines with
the Gaussian distributions that best fit the data.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_26" o:spid="_x0000_i1030" type="#_x0000_t75" style='width:367.5pt;
 height:400.5pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="BNW_workflow_test_files/image017.png" o:title="" croptop="11774f"
  cropbottom="5139f" cropleft="15267f" cropright="16897f"/>
</v:shape><![endif]--><![if !vml]><img width=490 height=534
src="BNW_workflow_test_files/image018.jpg" v:shapes="Picture_x0020_26"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>To further
investigate the structure of the network, users can select <u>Display Structure
Matrix</u> in the left menu. This will bring up a popup windoe containing a table
showing the confidence of each directed edge in the network after model
averaging. For this network, all of the nodes included in the network were
present in almost all high scoring networks, as the values of the edges are all
near 1. BNW displays all edges with a confidence greater than 0.5 in the
network structure. A second table in the window shows the structure matrix with
a 1 for edges included in the structure and 0 for edges that are not included. These
tables can be downloaded by clicking the download link.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_41" o:spid="_x0000_i1029" type="#_x0000_t75" style='width:358.5pt;
 height:158.25pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="BNW_workflow_test_files/image033.png" o:title="" croptop="19224f"
  cropbottom="21354f" cropleft="10363f" cropright="12698f"/>
</v:shape><![endif]--><![if !vml]><img width=478 height=211
src="BNW_workflow_test_files/image020.jpg" v:shapes="Picture_x0020_41"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><b style='mso-bidi-font-weight:
normal'><span style='font-size:14.0pt;line-height:115%;font-family:"Arial","sans-serif";
mso-no-proof:yes'>3) </span></b><b style='mso-bidi-font-weight:normal'><span
style='font-size:14.0pt;line-height:115%;font-family:"Arial","sans-serif"'>Make
predictions with the network structure<o:p></o:p></span></b></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'>BNW can be used to both
predict the values of variables in a network given known evidence and to
investigate how the network might change in response to interventions. Users
can switch between evidence and intervention modes by selecting the proper
button on the top of the page containing the network structure.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'>For example,
suppose we want to use the model to make predictions for a new sample that was
not in the original dataset. We know that this sample was from an individual
with Genotype=2 and want to predict the values of the genes and phenotype for
the individual. To enter this evidence in the network, we simply click on the
Genotype node in the network and enter 2 in the popup window. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'>The figure below shows the
changes in the network after entering this evidence.The node for which evidence was entered now contains a red outline and all of the nodes in the network now contain both blue and red lines. The blue lines show the original values of the distributions, while the red lines shown the predicted values given the
evidence. In this case, knowing that the individual had Genotype=2 would cause
us to predict that the expression of Gene1 and the value of the phenotype would
be above average, while the expression of Gene2 and Gene3 would be decreases.
The specific changes in the predicted values can be investigated by hovering
over the nodes and observing the values at the peaks of the distribution.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_29" o:spid="_x0000_i1028" type="#_x0000_t75" style='width:352.5pt;
 height:464.25pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="BNW_workflow_test_files/image019.png" o:title="" croptop="5662f"
  cropbottom="3557f" cropleft="15900f" cropright="17569f"/>
</v:shape><![endif]--><![if !vml]><img width=470 height=619
src="BNW_workflow_test_files/image022.jpg" v:shapes="Picture_x0020_29"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>Evidence
can be entered in more than one node in the network. For example, the figure
below shows predicted values for the network after entering evidence for both
Genotype and Gene1.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_32" o:spid="_x0000_i1027" type="#_x0000_t75" style='width:291.75pt;
 height:414pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="BNW_workflow_test_files/image021.png" o:title="" croptop="9437f"
  cropbottom="5850f" cropleft="16919f" cropright="22117f"/>
</v:shape><![endif]--><![if !vml]><img width=389 height=552
src="BNW_workflow_test_files/image024.jpg" v:shapes="Picture_x0020_32"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>To use the
network to predict the results of intervention, select the proper button on the
top of the window to enter intervention mode. While evidence simply changes the
predicted values of the other nodes in the network, intervention has a larger
effect, as it removes the dependence of the intervened network on its parents
and changes the network structure. <o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_35" o:spid="_x0000_i1026" type="#_x0000_t75" style='width:295.5pt;
 height:417.75pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="BNW_workflow_test_files/image023.png" o:title="" croptop="9437f"
  cropbottom="5375f" cropleft="16718f" cropright="21976f"/>
</v:shape><![endif]--><![if !vml]><img width=394 height=557
src="BNW_workflow_test_files/image026.jpg" v:shapes="Picture_x0020_35"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'>The figure
below shows the changes in the network in BNW after intervention on Gene1 that
results Gene1 having a low value. The predictions of the network after this
intervention could be compared with experiments that prevent Gene1 from being
expressed. Intervention that decreases Gene1 is predicted to result in an increase
in Gene2 and a decrease in the phenotype. Note that Genotype and Gene3, which
are not descendents of Gene1, are not affected by the intervention.<o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><!--[if gte vml 1]><v:shape
 id="Picture_x0020_38" o:spid="_x0000_i1025" type="#_x0000_t75" style='width:297.75pt;
 height:406.5pt;visibility:visible;mso-wrap-style:square'>
 <v:imagedata src="BNW_workflow_test_files/image025.png" o:title="" croptop="9382f"
  cropbottom="6755f" cropleft="16815f" cropright="21642f"/>
</v:shape><![endif]--><![if !vml]><img width=397 height=542
src="BNW_workflow_test_files/image039.jpg" v:shapes="Picture_x0020_38"><![endif]><o:p></o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif";mso-no-proof:yes'><o:p>&nbsp;</o:p></span></p>

<p class=MsoNormal style='margin-right:307.5pt'><span style='font-size:12.0pt;
line-height:115%;font-family:"Arial","sans-serif"'><o:p>&nbsp;</o:p></span></p>

</div>

</body>

</html>
